app-api/src/main/java/org/openmrs/eip/app/management/entity/ReceiverSyncStatus.java

```diff
@@ -64,8 +56,7 @@ public void setLastSyncDate(Date lastSyncDate) {
 		this.lastSyncDate = lastSyncDate;
 	}
 	
-	@Override
 	public String toString() {
-		return "ReceiverSyncStatus {siteInfoId=" + siteInfoId + ", lastSyncDate=" + lastSyncDate + "}";
+		return "ReceiverSyncStatus {siteInfoId=" + this.siteInfo.getId() + ", lastSyncDate=" + lastSyncDate + "}";
```

Comment:
I think you can change to siteInfo=this.siteInfo here and let the SiteInfo.toString method print the site